integration_test/third_party_apps_data/applications/activemq/minimum_supported_version

```diff
@@ -0,0 +1,3 @@
+activemq:
```

Comment:
https://github.com/GoogleCloudPlatform/ops-agent/blob/941e640c4c59d7324f91db3591e4fb1de0e92ee5/integration_test/third_party_apps_data/applications/apache/metadata.yaml#L16
this contains a sample format
Below is the expected format
```
minium_supported_agent_version:
  logging: 2.6.0
  metrics: 2.9.0
```